packages/asset/src/components/Editor/Editor.tsx

```diff
@@ -84,6 +84,14 @@ export const Editor = (props: EditorProps): JSX.Element => {
       provider.awareness
     );
 
+    const initialLoad = () => {
+      if(props.room === "notes" && type.length === 0) {
+        type.insert(0, "Interview Notes");
+      }
+    };
+
+    provider.on("synced", initialLoad);
+
```

Comment:
I am concerned that the `synced` event may in-inadvertently call `initialLoad` outside of the initial load of the WS connection.

I think instead we want to add an event listener for the `status` event with a value of `connected` for the `status`, see https://github.com/yjs/y-websocket/blob/aa4220407bda51ab6282d1291de6493c136c2089/src/y-websocket.js#L186-L188.

```suggestion
    const onStatus = (event) => {
      if (event.status !=== "connected") {
        return;
      }
      
      if (type.length === 0) {
        type.insert(0, "Interview Notes");
      }
      provider.removeEventListener(onStatus);
    };

    if (props.room === "notes") {
      provider.addEventListener("status", onStatus);
    }
```

Reply:
As an aside, thanks for the PR! This is a much needed improvement to highlight the interview notes editor. It might also be good to invest in some UI improvements such as headers for the editors to highlight their use. Contributions are welcome for that as well!

Reply:
I've been thinking about this more, I do not think I made the correct suggestion to move from the `sync` event to the `status` event. `status` likely does not reflect the document state. Can you still please make the change to detach the event listener after being called?